src/core/core.datasetController.js

```diff
@@ -998,7 +1023,7 @@ export default class DatasetController {
 		data.splice(start, 0, ...elements);
 
 		if (me._parsing) {
-			meta._parsed.splice(start, 0, ...new Array(count));
+			meta._parsedRaw.splice(start, 0, ...new Array(count));
```

Comment:
can't do the parsing here at all, because start and count refer to elements, no to the raw data.

Reply:
Yeah, I'm realizing that this is causing a lot of tests to fail. A lot of code assumes these can be updated in sync at the same time.

I wonder if there is a simpler way to achieve this without touching the parsing, maybe even just a good guess at the size for estimating the number of points to keep